src/attributeformmodelbase.cpp

```diff
@@ -214,16 +214,15 @@ void AttributeFormModelBase::updateAttributeValue( QStandardItem* item )
   }
 }
 
-void AttributeFormModelBase::flatten( QgsAttributeEditorContainer* container, QStandardItem* parent, const QString& visibilityExpressions, QVector<QStandardItem*>& items )
```

Comment:
would need to rename the arg in the header too

Reply:
Yes. Thanks.